examples/src/lib.rs

```diff
@@ -48,10 +48,12 @@ impl From<&CommonArgs> for Config {
             Some(path) => Config::from_file(path).unwrap(),
             None => Config::default(),
         };
+        println!("ARGS mode: {:?} ", value.mode);
```

Comment:
This line should be removed since now it's printed in very examples, including throughput and latency tests.